index.js

```diff
@@ -998,13 +987,6 @@ Client.prototype._currentReconcile = function (callback) {
 
         self.state.reconcileStamp = underscore.now() + (self.state.properties.days * msecs.day)
         if (self.options.verboseP) self.state.reconcileDate = new Date(self.state.reconcileStamp)
-
-        self._updateRules(function (err) {
-          if (err) self._log('_updateRules', { message: err.toString() })
-
-          self._log('_currentReconcile', { delayTime: msecs.minute })
-          callback(null, self.state, msecs.minute)
```

Comment:
This callback (to ledger.onCallback) looks like it has to be preserved to get the run loop to eventually end.